### PR TITLE
Page types other

### PR DESCRIPTION
**Pull request by hamishwillee**
HTML, HTTP, CSS, Glossary, Landing page - fix up the page templates to use the status rather than tags.
Also fix up all linked example pages to have the correct tags/status as well.

Does not include SVG as that isn't working yet.

### CODE REVIEW

File: files/en-us/web/javascript/index.md [hunk 1]
```diff
@@ -1,13 +1,7 @@
...
 title: JavaScript
 slug: Web/JavaScript
-page-type: guide
-tags:
-  - JavaScript
-  - Landing
-  - Landing page
-  - Learn
-  - "l10n:priority"
+page-type: landing-page
```
Comment (hamishwillee) [inline files/en-us/web/javascript/index.md:4]: FYI @Josh-Cena - landing page type is landing-page, so I changed this for JavaScript here.

Comment (Josh-Cena) [reply files/en-us/web/javascript/index.md:4]: This was @wbamberg's choice I believe. It feels like `landing-page` is for pages with mostly DLs while this has a large explanatory paragraph. But it's fine for me both ways.

Comment (hamishwillee) [reply files/en-us/web/javascript/index.md:4]: IMO probably landing page slightly better on basis that it's a major entry point whereas lots of things are "guide". So if we were to use the page type for formatting, I'd expect entry point to trump "run of the mill guide page". Let's see if @wbamberg complains, but I'm happy to change it,

**Review by teoli2003 — APPROVED** [review]
Approving but not merging. I was surprised to see tag removal bits included here, but they are fine.

Feel free to merge when you feel it is ok.

**Comment by hamishwillee** [comment]
I think it is OK now :-) The tag removal is opportunistic - not much point having links to other docs that show "the right way", if they still have tags. Generally the tags aren't used at all in guide, so not worried.

Thanks for the review.

File: files/en-us/web/javascript/index.md (hunk 1 above)
Comment (wbamberg) [reply files/en-us/web/javascript/index.md:4]: I agree `landing-page` is right for this. I think `landing-page` is right for pages that are primarily nav pages. It's fine (and usually expected) for landing pages to have some intro prose content explaining what the thing is (and if we ever standardised landing page structure we'd definitely have an affordance for this), but its main function is to orient someone and point them to further docs.

**Pull request merged**